src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/LeggedOdometry.h

```diff
@@ -104,11 +99,15 @@ class LeggedOdometry : public FloatingBaseEstimator
     /**
      * Get the index of the frame currently in contact used for the legged odometry computations
      * @return Index of the frame in contact, invalid frame index if no contact.
-     *
-     * @note reset and advance estimator to get updated estimator output
      */
     int getFixedFrameIdx();
 
+    /**
+     * Get the psoe of the current fixed frame in the inertial frame
+     * @return pose of the fixed frame
+     */
+    manif::SE3d getFixedFramePose() const;
```

Comment:
I think you can return a const ref

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/282/commits/eb377f39ab69284c173bcb8c3a428eb531ca575c